src/components/Results/ArticleSummary.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Grid, Typography, makeStyles } from "@material-ui/core/";
+import "../../App.css";
```

Comment:
@abirdonwire Does app.css need to be re-imported? I thought it'd be 'applied' by app.js and also apply to any child components?

Reply:
Hmm, I don't know! I'll investigate